platforms/web/src/containers/Root/Root.tsx

```diff
@@ -44,9 +52,7 @@ const DemoContentLoader = ({ query }: { query: BootstrapData }) => {
   return (
     <>
       {/* Show the error page when error except in demo mode (the demo mode shows its own error) */}
-      {!IS_DEMO_OR_PREVIEW && error && (
-        <ErrorPage title={error?.payload?.title} message={error?.payload?.description} error={error} helpLink={error?.payload?.helpLink} />
-      )}
+      {!IS_DEMO_OR_PREVIEW && error && renderError(error)}
```

Comment:
I would prefer to use a component here to make it consistent.

Reply:
Do you suggest creating another error component for handling `Error | AppError`, or to do that inline (2x)?

Or change `renderError` to `<BootstrapError error={error} />` and define `BootstrapError` in `Root.tsx`?

Reply:
Done!